rotkehlchen/exchanges/binance.py

```diff
@@ -680,6 +681,73 @@ def _query_margined_futures_balances(
 
         return balances
 
+    def _query_pools_balances(
+            self,
+            balances: DefaultDict[Asset, Balance],
+    ) -> DefaultDict[Asset, Balance]:
```

Comment:
Please add a docstring here. And perhaps in the other balance partial queries function. Mention that they may raise `RemoteError`

Reply:
Oh sorry! I scrolled all of them but my mind might have confused it with the start comment of the review :(